dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -37,7 +37,7 @@ private constructor(
         val serviceName: String,
         val envName: String,
         val trackGestures: Boolean = false,
-        val viewTrackerStrategy: ViewTrackerStrategy = ViewTrackerStrategy.NONE
+        val viewTrackingStrategy: ViewTrackingStrategy = ViewTrackingStrategy.NONE
```

Comment:
Didn't we choose to remove this enum ?

Reply:
Everything was refactored and removed in my next PR.